be/src/service/starrocks_main.cpp

```diff
@@ -271,13 +272,16 @@ int main(int argc, char** argv) {
         LOG(INFO) << "StarRocks BE HeartBeat Service started correctly.";
     }
 
+    Aws::SDKOptions aws_sdk_options;
+    Aws::InitAPI(aws_sdk_options);
+
     while (!starrocks::k_starrocks_exit) {
         sleep(10);
     }
-
     daemon->stop();
     daemon.reset();
 
+    Aws::ShutdownAPI(aws_sdk_options);
```

Comment:
use RAII for calling this function automatically?

Reply:
Great. I'll fix that.

Reply:
Well, A second thought, maybe this is ok, because

- When we are here, we are already in finalized stage. It's okay to not calling `shutdown`
- This way is much more simple than RAII, and it's explicit.
- And if we find there are some resource leaks, we can fix then.

Let's start with simple, and fix when necessary.